governance/process.md

```diff
@@ -69,10 +82,11 @@ from each company should vote. Members will vote by leaving a comment in the
 Pull Request to indicate their vote for or against. Members will have a week
 after the vote begins to leave their vote. Quorum is taken to be 2/3 the number
 of companies who have been active in the past month (via issue comment or 
-meeting attendace as recorded in the public minutes).
-
+meeting attendance as recorded in the public minutes).
 
-1. **Support the project going forward.** Some projects require ongoing
-support. When work is completed, the body of the Pull Request should specify if
-it's expected to require on-going support and if so, who will maintain it.ccept
+1. **Support the project going forward.** Some projects require sustainment and
+   maintenance to ensure continued relevance for the community.  When work is 
+completed, the body of the Pull Request should specify if it's expected to require
```

Comment:
do you mean Pull Request description?  I would think there would be a new proposal (issue) with a description of the expectation/plan/idea for on-going work